code/lib/cli/src/generators/baseGenerator.ts

```diff
@@ -279,9 +285,9 @@ export async function baseGenerator(
       : extraPackages;
 
   const allPackages = [
-    'storybook',
+    installStorybookPackage ? 'storybook' : undefined,
     getExternalFramework(rendererId) ? undefined : `@storybook/${rendererId}`,
-    ...frameworkPackages,
+    ...(installFrameworkPackages ? frameworkPackages : []),
```

Comment:
@kasperpeulen I will need to roll-back this optional installation of the `storybook` package to ensure this https://github.com/storybookjs/storybook/pull/28310 works.

I'm going to just proceed, but I'd like to evaluate with you on what to do about this.

Reply:
@tobiasdiez said that it works a bit different in the nuxt world

> The setup with nuxt is a bit different. In the nuxt world, you normally install a so-called module. The task of the module is to interface with whatever external library is used under the hood, as well as shipping the library (i.e. it is declared as a dependency).

> For example, the tailwind module for nuxt (https://github.com/nuxt-modules/tailwindcss) declares tailwindcss as a dependency (https://github.com/nuxt-modules/tailwindcss/blob/main/package.json#L56). Similarly, in the storybook module we already pull-in storybook as the dependency: https://github.com/nuxt-modules/storybook/blob/30cb7036f1bfc5bc50b11b6b7c3857bcf23886b6/package.json#L81 Thus users don't need to add the storybook packages themselves in their package.json.

Not sure how that would affect that in 8.2 `storybook` is required as a peer dep though. I guess we should test it to find out.

Reply:
This shouldn't be a problem for us. The nuxt module can pull-in whatever top-level storybook dependency is appropriate. In fact, we do this already for the `storybook` package: https://github.com/nuxt-modules/storybook/blob/d1bd15a990851bbcee53822b09acd046a657fd22/packages/nuxt-module/package.json#L50

@ndelangen what issues do you expect?